### ISSUE TITLE
Blocked

### ISSUE THREAD

**Issue by ecliptica777** [rainforestapp/tester-chrome-extension#578]
Hello,

Recently I saw in the terminals: "Page not found" in the start page (instead welcome to rainforest terminal). I continued to run the test as usual. Copied the link from the instructions in the terminal and etc. 

I think when I see "Page not found" in the start page, the rainforest system automatically considers the test a failure. However, maybe I'm wrong, I wrote in support of Rainforest.

<img>https://pp.vk.me/c626628/v626628959/280ec/ieIWmGnuUxw.jpg</img>

**Comment by nerdatwork** [comment]
Is that your profile ? If so you got a job after being blocked ?

**Comment by ecliptica777** [comment]
nerdatwork, yes it's my profile. I can't got any job when I blocked. I hope soon my account will be unlocked. Otherwise, very sorry...

**Comment by peddlerguy** [comment]
Maybe I'm misunderstanding, but are you saying that your account was blocked because when you got "page not found" in the terminal when it first loaded, you went ahead and entered the url from the instructions to load the test page and continued on?

I thought that was what we were supposed to do?  I read it somewhere - because that initial page that usually says "welcome to rainforest terminal" isn't actually part of the site we're testing?

**Comment by nerdatwork** [comment]
@ecliptica777 Accounts once blocked are rarely unblocked and it clearly says you fail to follow instructions so you wont be able to do any more jobs.

**Comment by ecliptica777** [comment]
@nerdatwork peddlerguy, I assume that's one of the reasons blocking of my account. Indeed, it is not part of the test. Just strange to me that my account is blocked. I usually always followed the instructions. And the account was blocked just after the tests, where the starting page said: "Page not found". However, maybe I'm wrong.

@nerdatwork well.. so.. I'm a bad tester :(

**Comment by sweeneyapps** [comment]
@ecliptica777 it could be another reason, you ll have to talk with RF about it.   It can be anything like opening an youtube link or go on facebook inside the Virtual Machine.   I know of that one tester got banned for doing that.

**Comment by uangonline** [comment]
@ecliptica777 I don't think that's the reason you get blocked. It must be other reason. If you already contacted the suport team, wait for their respond to know the real reason.

**Comment by ecliptica777** [comment]
@sweeneyapps I used the terminal only for testing. Facebook only if specified in the instructions. 

I will wait for a response from support Rainfoest.

**Comment by euparkeria** [comment]
It says you failed too many times, I doubt this has anything to do with "Page not found". And frankly your bug found rate is very low.
